lib/src/Cookie.cc

```diff
@@ -77,3 +77,20 @@ std::string Cookie::cookieString() const
     ret.append("\r\n");
     return ret;
 }
+
+Cookie::SameSite Cookie::convertString2SameSite(const std::string& sameSite)
+{
+    if (sameSite == "Lax")
+    {
+        return Cookie::SameSite::kLax;
+    }
+    else if (sameSite == "Strict")
+    {
+        return Cookie::SameSite::kStrict;
+    }
+    else if (sameSite == "None")
+    {
+        return Cookie::SameSite::kNone;
+    }
+    return Cookie::SameSite::kNull;
```

Comment:
@an-tao I think we should add a `LOG_WARN << sameSite << " is not a valid SameSite polity";` warning. Your thoughts?